server/scripts/feed-sites.js

```diff
@@ -3,7 +3,7 @@ require('dotenv').config({ path: '../.env' });
 const readXlsxFile = require('node-xlsx').default;
```

Comment:
was this file missed for the import update?

Reply:
There's certain things I left as requires for now, due to issues importing them - this might have been one of them, can't remember. I'll check later.

Reply:
Resolved in fbc7c23.